backend-py/Dockerfile

```diff
@@ -1,7 +1,11 @@
 FROM python:3.9 as backend-py
```

Comment:
probably want to use one of the slim variants rather than the bulky i mage

Reply:
I left it out of this PR to reduce changes.